boringtun/src/noise/mod.rs

```diff
@@ -6,8 +6,6 @@ pub mod handshake;
 pub mod rate_limiter;
 
 mod session;
-#[cfg(test)]
-mod tests;
```

Comment:
These tests, it turns out, never worked in the first place, and were being quietly ignored until I started using that feature for expensive tests.